public/walkthroughs/my-custom-walkthrough/walkthrough.adoc

```diff
@@ -38,13 +58,31 @@ Check one more thing
 [type=verificationFail]
 Heck 'em!
 
+This is a Che route, I hope: {che-url}
+
+[type=taskResource]
+=== Some resources
+
+Woop woop doop doop
```

Comment:
Might be better to have the example text explain what this tag & area are for?

Reply:
Good point. I will clean up this document  so each section explains the feature.